doc/releases/migration-guide-4.0.rst

```diff
@@ -538,10 +552,17 @@ JWT (JSON Web Token)
 ====================
 
 * By default, the signature is now computed through PSA Crypto API for both RSA and ECDSA.
-  The newly-added :kconfig:option:`CONFIG_JWT_USE_LEGACY` can be used to switch
-  back to previous libraries (TinyCrypt for ECDSA and Mbed TLS for RSA).
-  The conversion to the PSA Crypto API is being done in preparation for the
-  deprecation of TinyCrypt. (:github:`78243` and :github:`43712`)
+  (:github:`78243`). The conversion to the PSA Crypto API is part of the adoption
+  of a standard interface for crypto operations (:github:`43712`). Moreover,
+  following the deprecation of the TinyCrypt library (:github:`79566`) usage
+  of TinyCrypt was removed from the JWT subsystem (:github:`79653`).
+
+* The following new symbols were added to allow specifying both the signature
+  algorithm and crypto library:
+
+  * :kconfig:option:`JWT_SIGN_RSA_PSA` (default) RSA signature using PSA Crypto APIs;
+  * :kconfig:option:`JWT_SIGN_RSA_LEGACY` RSA signature using Mbed TLS;
+  * :kconfig:option:`JWT_SIGN_ECDSA_PSA` ECDSA signature using PSA Crypto APIs.
```

Comment:
That's correct. I didn't intentionally add the previous symbols (those introduced in #78243) because they were never part of any release if this PR had make it to be merged before 4.0 code freeze. However this is not the case anymore, so yes, I need to update the documentation as well :)

Reply:
Maybe there's still a small chance for this to go in for 4.0. Dunno. But also, the original JWT Kconfig options that were removed, those that you did not introduce yourself, are they documented in the migration guide?

Reply:
> Maybe there's still a small chance for this to go in for 4.0

I don't know either. If not, then I will address this chance when the new migration-guide document will be added to the codebase.

>  But also, the original JWT Kconfig options that were removed, those that you did not introduce yourself, are they documented in the migration guide?

#78243 documented new symbols [added there](https://github.com/zephyrproject-rtos/zephyr/pull/78243/files#diff-ada0effa18bf591a10f0e71db409689e026c8c35cd790a1b0026d25cac005535R364), but no, this one lacks documentation about the removal